cli/src/main/java/com/devonfw/tools/ide/version/VersionPhase.java

```diff
@@ -12,7 +12,7 @@ public enum VersionPhase implements AbstractVersionPhase {
   REVISION(Boolean.TRUE, "revision", "rev"),
 
   /** A snapshot version from development (e.g. "-SNAPSHOT" suffix in maven). */
-  SNAPSHOT(Boolean.FALSE, "snapshot", "dev"),
+  SNAPSHOT(Boolean.FALSE, "beta-snapshot", "snapshot", "dev"),
```

Comment:
is the JUnit red without this change?
So `alpha-SNAPSHOT` or `rc-SNAPSHOT` would also not work?

This remark is not to block this PR. I am fine to merge and release your great fix.
I just want to tackle if we might later want to add another improvement PR if still some things are odd in this area...

Reply:
Created story #1178